tests/robot-tests/tests/admin_and_public_2/bau/publish_release_and_amend.robot

```diff
@@ -170,6 +170,45 @@ Add embedded dashboard to third accordion section
     user waits until h1 is visible    DfE Analytical Services R-Shiny data dashboard template (h1)    90
     unselect frame
 
+User navigates to Data blocks page to edit block
+    [Documentation]    EES-3009
+    user clicks link    Data blocks
+    user waits until h2 is visible    Data blocks    %{WAIT_SMALL}
+
+Edit data block
+    #just updating the table title and source
+    user waits until table is visible
+    user clicks link    Edit block    css:tbody > tr:first-child
+    user waits until h2 is visible    ${DATABLOCK_NAME}
+    user waits until h2 is visible    Data block details
+
+    user enters text into element    id:dataBlockDetailsForm-name    ${DATABLOCK_NAME}
+    user clears element text    id:dataBlockDetailsForm-heading
+    user enters text into element    id:dataBlockDetailsForm-heading    Updated dates table title
+    user clears element text    id:dataBlockDetailsForm-source
+    user enters text into element    id:dataBlockDetailsForm-source    Updated dates source
+
+    user clicks button    Save data block
+    user waits until page contains button    Delete this data block
+
+Navigate to the 'Content' page
+    user clicks link    Content
+    user waits until h2 is visible    ${PUBLICATION_NAME}
+    user waits until page contains button    Add a summary text block    %{WAIT_SMALL}
+
+Verify data block is updated correctly
+    #checking if data block cache has been invalidated by verifying the updates on the block
+    user scrolls to accordion section content    Dates data block    id:releaseMainContent
+    user opens accordion section    Dates data block    id:releaseMainContent
+    ${section}=    user gets accordion section content element    Dates data block    id:releaseMainContent
+
+    user checks chart title contains    ${section}    Updated dates table title
+    user clicks link by visible text    Table    ${section}
+    user waits until parent contains element    ${section}
```

Comment:
I've seen this written in other places like:

```
    user waits until element contains    testid:dataTableCaption
    ...    Admission Numbers for 'UI test subject' in Nailsea Youngwood and Syon between 2005 and 2017
```

which cuts down a bit of xpath.  Consider using this way instead?

For the future, we should probably have another common keyword for this, like:

`user waits for data table caption`

so we don't have to repeat this pattern all over the place :)

Reply:
Not suggesting we need to do this as part of this PR.